### ISSUE TITLE
bug: accessibility of wizard needs a bit of work.

### ISSUE THREAD

**Issue by marrie** [Tarsnap/tarsnap-gui#82]
Hello there. I'm pretty new to the tarsnapgui but I' here to report an issue.

I'm a visually impaired user who is using the GUI. I'll learn the command line interface as well but for now I'm using the gui. I  notice in the setup wizard there are a set of 5 radio buttons including one that is dimmed. These haveno help tags for voiceover users. Voiceover is the screenreader that ships with mac os and reads to me what is on the screen and allows I and others  to interact with our macs. I've attached a screenshot of the effected area. I hope this helps a bit.
![screenshot showing the area of the wizard effected. Notice the unlabeled radio buttons.](https://cloud.githubusercontent.com/assets/3614487/15658827/14cf54e8-2676-11e6-8e12-2e037d66a581.png)

**Comment by marrie** [comment]
Ah thanks. Just to add in case I forgot, the radio buttons don't have help tags on them. This would be helpful, or at least a step sward progress.

**Comment by shinnok** [comment]
Adding the tooltips is easy. Graham, do you want to do more then that?

The radio buttons are there to show the overall progress of the setup as well as provide a means to go back to a previous step. As Graham said, they won't be needed for most people, apart from the case where they need to correct a mistake in a previous step, this is partly the reason why I disabled tab focus for them.

**Comment by marrie** [comment]
Tabbing is not necessary as long as we can read the actions those radio buttons will invoke. I would go to www.applevis.com and look at their wonderful documentation that is someware on the site regarding accessibility and how to implement it on mac. I can't help with windows but you can test the app using nvda found at http://www.nvda-project.org and talk to those developers.

hope that helps a bit.

**Comment by gperciva** [comment]
@shinnok Yes, I'm planning on installing some screen-reading software and do a full review of the gui accessibility.  I'm sure there's more issues than just the two that have been pointed out so far, so I might as well prepare a bunch of PRs.

**Comment by marrie** [comment]
One for of advice. Turn off your screen when working with the app and use only the keyboard to get around. I tell some of the sighted devs to do this and they are surprised about stuffthey find in their own app. 

hope that helps a bit.

**Comment by shinnok** [comment]
@marrie I'd like to follow up on this.

We didn't approach this issue directly yet, but there have been many UI, wording & functional improvements since v0.8 that would make the application easier to use and navigate on the whole. I'd like to give you a heads up on the fact that we're also going to release v1.0 this month and maybe you can take it for a spin and come back with some feedback. I can then evaluate this situation and maybe devise a plan for tackling this.
